### PR TITLE
Adding the create call failed event

### PR DESCRIPTION
**Pull request by v-dharmarajv**
Api View : https://spa.apiview.dev/review/85dcaed7e707442da4ff62e68c2ab36a?activeApiRevisionId=a2786a021b8e4df193e9bf2be713a21e&diffApiRevisionId=c7d7c14b688242d6bda51dc4c7d34540

Added the createcallfailed event to beta4 version

### CODE REVIEW

File: sdk/communication/Azure.ResourceManager.Communication/src/Azure.ResourceManager.Communication.csproj
Comment (minwoolee-msft) [inline sdk/communication/Azure.ResourceManager.Communication/src/Azure.ResourceManager.Communication.csproj:1]: is this file intended for change?

Comment (v-vdharmaraj) [reply sdk/communication/Azure.ResourceManager.Communication/src/Azure.ResourceManager.Communication.csproj:1]: Actually no, thanks for catching. I have updated it.

**Review by minwoolee-msft — APPROVED** [review]
